packages/nodes-base/package.json

```diff
@@ -749,7 +749,7 @@
     "@types/tmp": "^0.2.0",
     "@types/uuid": "^8.3.2",
     "@types/xml2js": "^0.4.3",
-    "eslint-plugin-n8n-nodes-base": "^1.11.0",
+    "eslint-plugin-n8n-nodes-base": "^1.11.1",
```

Comment:
this shouldn't be making 16k changes in the lock file. something seems off.

Reply:
Installing with `^1.11.0` was pulling `1.11.0` despite `1.11.1` being released. I'll recheck.